Installing-and-Configuring/Building-Heads/t430s.md

```diff
@@ -0,0 +1,84 @@
+---
+layout: default
+title: Lenovo T430s
+nav_order: 1
+permalink: /t430s-building/
+parent: Step 1 - Building Heads
+grand_parent: Installing and configuring
+---
+
+Lenovo T430s
+====
+
+<!-- markdownlint-disable MD033 -->
+<details open markdown="block">
+  <summary>
+    Table of contents
+  </summary>
+  {: .text-delta }
+1. TOC
+{:toc}
+</details>
+<!-- markdownlint-enable MD033 -->
+
+This guide only documents build options you might be interested in, which are:
+If you are not interested in those options follow the [General Building](/general-building/) guide.
+
+- Setting an arbitrary MAC address.
+- Change some BIOS options (like swap the fn and ctrl-key)
+
+# Preparation
+
+Follow the [General Building](/general-building/) until you have installed all dependencies and cloned the tree. _Do not_ execute `make BOARD=t430s-maximized` yet!
+
+# Edit build (optional)
+
+## Change MAC of Ethernet NIC (via Hexeditor)
+
+
+The ethernets NIC firmware is located at: `blobs/t430s/gbe.bin`
+
+The MAC address appears twice in there at the offset: `0x0` and `0x1000` and is set to `00:de:ad:c0:ff:ee`.
+
```

Comment:
@Siproqu That is confirmed to work? 
I tested this on x230 which [confirmed to not work](https://github.com/osresearch/heads/issues/307#issuecomment-1019352022)

Reply:
@Siproqu Can you generated a GBE blob by [modifying the bincfg /gbe-82579LM set](https://github.com/coreboot/coreboot/blob/master/util/bincfg/gbe-82579LM.set#L34) using the same MAC address you used to test this?  Then use a tool like `vbindiff` to show the differences between the blob you just created and the original that uses the MAC  `00:de:ad:c0:ff:ee`.  As the validation is only a checksum, it is quite possible that the MAC you used and `00:de:ad:c0:ff:ee` both generate the same checksum but for most other MAC addresses this would not work.

Reply:
@tlaurion It's been a while since I've done that (and since then a default coreboot is running on the board), but as far as I remember it worked.

@Thrilleratplay Will do it. And fix the guide if necessary. But Heads is a little low in the priority list at the moment. Please give me some time.

Reply:
I could confirm that generating the gbe blob with bincfg works fine. I'll change the wiki entry in the next days.
Probably did something wrong when I simply changed the MAC via hexeditor. Anyways.

Will also update https://github.com/osresearch/heads/pull/1019 and https://github.com/osresearch/heads/pull/723 (if still needed?).

Sorry that everything takes so long and so many mistakes happen. Heads is currently very low on the priority unfortunately.